### DISCUSSION TITLE
velero client not able to access S3 backend after upgrade to v1.9.1

### DISCUSSION THREAD

**Discussion by GiuseppeLoRe** [vmware-tanzu/velero#5328] [Community support Q&A] [answered]
Hi, 
I have upgraded my velero instance from v.17 to 1.9. Most of it went fine, backup operations are still working.
The only main issue is now that when I type "velero backup describe --details" or "velero backup logs" it hangs, like if there was something misconfigured relative to the S3 backend.
Could someone help in debugging/fixing this?
Thanks in advance,

Giuseppe

**Comment by GiuseppeLoRe** [comment]
Just adding, I tried to run the velero client from different servers, and also tried again the old version (1.7). No help.

**Comment by Lyndon-Li** [comment, marked as answer]
This looks like not a generic problem, so please open an issue with Velero log bundle attached. Thanks.

**Reply by GiuseppeLoRe** [reply]
Thanks for your answer. I have created #5357.
